chingu-voyages/v52-tier3-team-38
# Project Structure + Navbar + Header

# Description
This PR introduces the Header and Navbar components to our app's layout. It also integrates Bootstrap, React-Bootstrap, and react-icons to enhance the user interface and simplify styling.

## Related to the following Jira/Trello issue
Header card
Navigation Bar card


## Main changes explained:
- Created a components folder: Organized the structure for reusability and scalability.
- Added Header and Navbar components:
- Header: Enhanced with interactive buttons for better usability.
- Navbar: Includes a stylish GitHub icon (thanks to react-icons! 🎨).
- Installed dependencies:
- Bootstrap and React-Bootstrap for consistent styling.
- react-icons for adding sleek icons.
- Updated layout.js to include the new components for seamless integration.

## How to test:
1. Switch to this branch: git checkout Ross/feature/navbar
2. Run the app locally:
```
npm install  
npm run dev
```  
3. Navigate to the app’s main page and verify:
- The Header displays the interactive buttons as expected.
- The Navbar includes the GitHub icon and functions properly.

## Screenshots or videos of changes:
![image](https://github.com/user-attachments/assets/f7bc04b6-8ee9-46b8-bfa1-443f36a5ee9a)

![image](https://github.com/user-attachments/assets/a453e55e-d346-46ba-aecc-6c8af99c8bbe)



## Note:
This is still a work in progress. I plan on doing more, and working on the landing page as well.
